### DISCUSSION TITLE
Is it possible to completely ignore handwritten text?

### DISCUSSION THREAD

**Discussion by 32h3basfl2** [PaddlePaddle/PaddleOCR#13608] [Q&A]
I have a document with handwritten notes. However I want to recognize only printed text, not those handwritten notes.

For example:
![image](https://github.com/user-attachments/assets/018a2fcc-55ad-4153-bac2-6857046b5d87)
I want only the printed field from left (e.g.: Phone Number), not the handwritten text (such as 555-0100)

For now, the handwritten text may or may not be detected, but I want to always exclude it from detection. Is it possible?

Reversely, if it is possible to detect ONLY handwritten text but NOT printed text, the logic will still apply if possible.

**Comment by SWHL** [comment]
It could be. You have to train the detect model that only detects the printed text.
